src/dune_pkg/opam_solver.ml

```diff
@@ -101,10 +101,10 @@ module Context_for_dune = struct
     { repos; version_preference; local_packages; solver_env }
   ;;
 
-  type rejection = Unavailable
+  type rejection = [ `Unavailable ]
```

Comment:
Why is this changed?

Reply:
I had tried something but reverted it later, looks like I missed this.